app/mails/models.py

```diff
@@ -34,7 +37,7 @@ class Recipient(models.Model):
 
 
 class Due(models.Model):
-    mail = models.ForeignKey(Mail, on_delete=models.CASCADE)
+    mail = models.ForeignKey(Mail, on_delete=models.CASCADE, related_name="dues")
```

Comment:
This needs a migration, as it will cause warnings otherwise. Just run `manage.py makemigrations`